src/Components/Bookmark.jsx

```diff
@@ -0,0 +1,150 @@
+import React,{useEffect, useState} from "react";
+import "../assets/css/Bookmark.css";
+
+
+function DailyForecast({day, forecast}) {
```

Comment:
@joshi-kaushal 
If I understand you currently, multiple components should not be inside a single `.jsx` file?

Reply:
Yes, you are right.

Reply:
We can have multiple components in same file.

Reply:
Did not use prettier because it would have then created merge conflicts.

Reply:
The merge conflicts caused by prettier are comparatively simpler, just accept changes from your side unless you see a changed expression or statement.

Reply:
I am resolving this conversation as putting the components in one file or creating a different file for each of them is one and the same. Also we preferred the former because the the components were to be only used by Bookmark Container.
 @joshi-kaushal @arpitBhalla

Reply:
@arpitBhalla unless the function is memoized in some way, I guess it does cause the entire file to rerender. I might be wrong.

Reply:
Nope